packages/expo-camera2/src/UnavailabilityError.ts

```diff
@@ -0,0 +1,13 @@
+import { Platform } from 'react-native'
+
+export default class UnavailabilityError extends Error {
```

Comment:
Why do you need to have this in `expo-camera2` instead of just importing from `@unimodules/core`?

Reply:
Leftover from being outside expo for a while 😉